src/utils/text.ts

```diff
@@ -33,7 +37,18 @@ export function calculateEditorColFromVimScreenCol(line: string, screenCol: numb
         }
 
         if (currentCharIdx >= line.length) {
```

Comment:
@ollien Here we cannot use width; what we need is character index

Reply:
Nice catch. I had a test failure on one of the double-wide unit tests but must have fixed it otherwise